.github/workflows/backend-api-pull-request.yml

```
@@ -12,64 +11,17 @@ on:
     paths:
       - "backend-api/**"
       - ".github/workflows/backend-api-pull-request.yml"
```

issue: These workflows should also be triggered if the constituent parts are altered. 

For exmaple: 
```suggestion
      - .github/workflows/backend-api-pull-request.yml
      - .github/workflows/job_ci-checks.yml
      - .github/workflows/job_test-suite.yml
```

It isn't a necessity for the ci-checks, however i consider it as such for build + deploy steps. For example if the `devplatform-upload-action` has been updated then I would want us to run the post-merge workflows where it is used. Similarly for changes in any of the dependencies of the pre-merge workflows. 

The additional benefits here is that we verify the workflows work so we aren't surprised by a breaking change when attempting to deploy a functional change.

---

ahhh good point